src/control/cmd/dmg/system.go

```diff
@@ -38,6 +38,7 @@ type SystemCmd struct {
 	LeaderQuery leaderQueryCmd       `command:"leader-query" alias:"l" description:"Query for current Management Service leader"`
 	MemberQuery systemMemberQueryCmd `command:"member-query" alias:"q" description:"Retrieve DAOS system membership"`
 	Stop        systemStopCmd        `command:"stop" alias:"s" description:"Perform controlled shutdown of DAOS system"`
+	Restart     systemRestartCmd     `command:"restart" alias:"r" description:"Perform restart of stopped DAOS system"`
```

Comment:
Should this be named "restart" or just "start"? Currently it's true that the only workflow is stop->restart, but it seems possible that we might want to have a way of starting the control plane without always automatically starting the data plane. In any case, from a user's perspective, I would assume that "restart" works to handle the entire stop/start process, not just starting again after a stop. There is strong precedent for this in the way init scripts and systemd units work.

I'm not suggesting that we need to support restart in addition to stop/start right now -- I think stop/start is fine for the purposes of this PR.